_posts/en/posts/2016-06-06-compact-blocks-faq.md

```diff
@@ -0,0 +1,142 @@
+---
+type: posts
+layout: post
+lang: en
+name: compact-blocks-faq
+id: en-compact-blocks-faq
+title: Compact Blocks FAQ
+permalink: /en/2016/06/06/compact-blocks-faq/
+version: 1
+---
+{% include _toc.html %}
+
+*Compact block relay* is a method of reducing the amount of bandwidth used to propagate new blocks to full nodes.
+
+## Summary
+
+Using simple techniques it is possible to reduce the amount of bandwidth necessary to propagate new blocks to full nodes when they already share much of the same mempool contents. Peers send compact block “sketches” to receiving peers. These sketchest include the following information:
+
+- The 80-type header of the new block
+- Shortened transaction identifiers (txids), that are encoded to prevent Denial-of-Service (DoS) attacks
```

Comment:
s/encoded/designed/

It makes it sounds like that's all they're doing. They primarily help identifying the transactions, but are specially designed to withstand certain DoS attacks.